tests/browser_test.py

```diff
@@ -1,8 +1,8 @@
+import os
 import unittest
-from octbrowser import __version__ as ob_version
-from octbrowser.browser import Browser
-
+from collections import deque
 import threading
+import urllib
```

Comment:
unfortunatelym urllib has been reworked in python3 and this can cause problems, but we can use six like this : 

``from six.moves import urllib`` for python 2 and 3 compatibility